docs/administration/configuration/configuring-access/index.md

```diff
@@ -24,8 +24,64 @@ Helm value accordingly.
 
 ## User registration via Single Sign-On
 
-Handling of users with Single Sign-On (SSO) is also possible. Currently,
-this is only available for CERN deployments via [`components.reana_ui.cern_sso`](https://github.com/reanahub/reana/tree/master/helm/reana)
+Handling of users with Single Sign-On (SSO) is also possible. Currently SSO is supported for CERN as well as for any third-party SSO provider which supports [Keycloak](https://www.keycloak.org/).
```

Comment:
"Handling of users" may have some negative connotations; perhaps say "User access through Single-Sign-On (SSO) authentication is also possible". (More neutral.)

Reply:
REANA currently supports CERN SSO as well as any third-party...